csharp/Platform.Collections/ICollectionExtensions.cs

```diff
@@ -17,23 +17,23 @@ public static class ICollectionExtensions
         /// <para>Проверяет коллекцию на пустоту.</para>
         /// </summary>
         /// <typeparam name="T">
-        /// <para>Collection element data type.</para>
+        /// <para>Collection elements data type.</para>
         /// <para>Тип данных элемента коллекций.</para>
         /// </typeparam>
         /// <param name="collection">
-        /// <para>Method takes an elements collection of <see cref="ICollection<T>"/> type.</para>
-        /// <para>Метод принимает колекцию элементов <see cref="ICollection<T>"/> типа.</para>
+        /// <para>Takes an elements collection of <see cref="ICollection<T>"/> type.</para>
+        /// <para>Принимает колекцию элементов <see cref="ICollection<T>"/> типа.</para>
         /// </param>
         /// <returns>
-        /// <para>Returns a <see cref="bool"/> type variable equal to false if the collection is empty else returns true.</para>
-        /// <para>Возвращает переменную типа <see cref="bool"/> равной false если коллекция пустая иначе возвращает true.</para>
+        /// <para>The <see cref="bool"/> type variable equal to false if the collection is empty else returns true.</para>
```

Comment:
Good

Reply:
`The <see cref="bool"/> type variable equal to false if the collection is empty else returns true.`->`A <see cref="bool"/> value that determines whether the collection is empty.`